tests/business.test.ts

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-import { Marqeta } from '../src';
+import { Marqeta } from '../src/index';
```

Comment:
these should be identical:

```typescript
import { Marqeta } from '../src'
```

and:

```typescript
import { Marqeta } from '../src/index'
```

Also, note that if we don't _need_ a semi-colon, then we don't add them.